ninja.lua

```diff
@@ -40,7 +40,7 @@ end
 
 local build_cache = {}
 
-local function add_build(cfg, out, implicit_outputs, command, inputs, implicit_inputs, dependencies, vars)
+function ninja.add_build(cfg, out, implicit_outputs, command, inputs, implicit_inputs, dependencies, vars)
```

Comment:
What are the reasons to promote the local function as method member?

Reply:
I'm fine to have simpler PR in preparation for future PR.
I just wanted to have the big picture.